springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/components/ComponentsScanner.java

```diff
@@ -9,13 +9,16 @@ public interface ComponentsScanner {
      * @param basePackage The root package to scan.
      * @return A set of found classes.
      */
-    Set<Class<?>> scanForComponents(Package basePackage);
+    default Set<Class<?>> scanForComponents(String basePackage) {
```

Comment:
Let's remove the argument completely form `scanForComponents` and leave it without arguments. Then we can also remove the `AsyncAciDocket` wirings from `AbstractChannelScanner` etc. and have `DefaultComponentScanner` ue the docket directly.